### PR TITLE
[16.0][mig] stock_picking_show_backorder: Migration to 16.0

### PR DESCRIPTION
**Pull request by dsolanki-initos**

### CODE REVIEW

**Review by CasVissers-360ERP — APPROVED** [review]
LGTM we have v15 running currently in a v16 production env

**Review by bosd — APPROVED** [review]

File: stock_picking_show_backorder/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Stock Picking Show Backorder",
-    "version": "15.0.1.0.1",
+    "version": "16.0.1.0.0",
```
Comment (rousseldenis) [inline stock_picking_show_backorder/__manifest__.py:6]: @dsolanki-initos Could you move version change in migration commit? Thanks

Comment (bosd) [reply stock_picking_show_backorder/__manifest__.py:6]: LGTM now

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**